README.md

```diff
@@ -1 +1,120 @@
-# helllo-microverse
\ No newline at end of file
+# helllo-microverse
+
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h3><b>Hello Microverse</b></h3>
+
+</div>
```

Comment:
- [ ] kindly reduce the number of titles you have in your Readme to make it more proffessional

Reply:
Thank you. @divinecharlotte I just did the changes and committed them for review